config/default/field.field.paragraph.civictheme_message.field_c_p_content.yml

```
@@ -0,0 +1,22 @@
+    - paragraphs.paragraphs_type.civictheme_message
+  module:
+    - text
+id: paragraph.civictheme_message.field_c_p_content
+field_name: field_c_p_content
+entity_type: paragraph
+bundle: civictheme_message
+label: Content
+description: ''
+required: false
+translatable: false
+default_value: {  }
```

YAMLlint (1.37.1)

[error] too many spaces inside empty braces

(braces)